tests/test_utils.py

```diff
@@ -182,7 +182,7 @@ def table_1() -> OrderedDict[str, Iterable]:
 
 
 @pytest.fixture
-def table_1_np_arrays() -> OrderedDict[str, Iterable]:
+def table_1_np_arrays() -> OrderedDict[str, Union[List, np.ndarray]]:
```

Comment:
This, and all other places this change was made in the test file, should be changed to `UnpackedArray` to match the current typing of the functions.